doc/sphinx/proof-engine/tactics.rst

```diff
@@ -3262,19 +3263,19 @@ the conversion in hypotheses :n:`{+ @ident}`.
      After that command, the expression :g:`(minus (S x) y)` is left untouched
      by :tacn:`simpl`, while :g:`(minus (S x) (S y))` is reduced to :g:`(minus x y)`.
 
-   + A special heuristic to determine if a constant has to be unfolded
-     can be activated with the following command:
+   .. todo: below: "unfolded" is equivalent to "apply a simplification step"?
```

Comment:
I will leave it as "unfolded".  How about the use of "constant"?  In your examples above, I wouldn't consider `Nat.add` a function.  But we can leave "constant" as-is for this PR if it's not a marginal improvement.

Reply:
"constant" or even "defined constant" mean any object that was introduced in the global context by a command like `Definition`. It is a "constant" because it represents a fixed term, unlike variables. And since it is a constant with a functional type, it is also a "function" (see the comments in my next review regarding this word). Once more, this should improve once we get a glossary.

Reply:
Perhaps you could provide a definition of "constant"?  We can put it somewhere now and maybe move it later.